test/create_parameters.jl

```diff
@@ -21,50 +22,17 @@ function cloud_microphysics_parameters(toml_dict)
     thermo_params = TD.Parameters.ThermodynamicsParameters{FT}(; pairs...)
     TP = typeof(thermo_params)
 
-    aliases = string.(fieldnames(CM.Parameters.CloudMicrophysicsParameters))
-    aliases = setdiff(aliases, ["thermo_params"])
+    aliases = string.(fieldnames(CMP.ModalNucleationParameters))
     pairs = CP.get_parameter_values!(toml_dict, aliases, "CloudMicrophysics")
-    return CM.Parameters.CloudMicrophysicsParameters{FT, TP}(;
+    modal_nucleation_params = CMP.ModalNucleationParameters{FT}(; pairs...)
+    MNP = typeof(modal_nucleation_params)
+
+    aliases = string.(fieldnames(CMP.CloudMicrophysicsParameters))
+    pairs = CP.get_parameter_values!(toml_dict, aliases, "CloudMicrophysics")
+
+    return CMP.CloudMicrophysicsParameters{FT, TP, MNP}(;
         pairs...,
         thermo_params,
+        modal_nucleation_params,
```

Comment:
This still puts all parameters into one struct right?

I was wondering if it would be better for GPU performance to have smaller separate structs? Or it doesn't make a difference?

Reply:
I'm not sure, I can ask Sriharsha.

Reply:
So far we have this: https://github.com/CliMA/CloudMicrophysics.jl/pull/194

So at least for now we pass in the whole struct, but we interact with it differently inside the functions. Such that we don't have to unpack the parameters by hand inside the functions.